### DISCUSSION TITLE
GoPro Labs Downloading Issue

### DISCUSSION THREAD

**Discussion by mtachannel** [gopro/labs#679] [Q&A]
Hello Gents, 

I am trying to download the Gopro labs in order to update and customise my naked 11 but when it comes to the process it is just stuck and nothing happens ,,,,, does anyone knows what is the issue !!!!

**Comment by dnewman-gpsw** [comment]
Do you have an earlier version of Labs on your HERO11?  Can you share you log file from MISC/qrlog.txt

**Comment by mtachannel** [comment]
Hello,

This is my first time to download the lab from the website.
I don’t have earlier version but the issue is actually related to the
GitHub server since when I click on the link related to GP 11 black to
download  the software ,,,, it takes ages to load then failure message will
appear afterwards.
I really don’t know what to do, I made even several researches on that
topic to figure out what is going on but apparently didn’t find any thing.
